test/txsim/sequence.go

```diff
@@ -43,8 +43,8 @@ type Operation struct {
 
 const (
 	// set default gas limit to cover the costs of most transactions
-	// At 0.001 utia per gas, this equates to 1000utia per transaction
-	DefaultGasLimit = 1000000
+	// At 0.1 utia per gas, this equates to 20000utia per transaction
+	DefaultGasLimit = 200000
```

Comment:
[naive question] I think the relationship may be: 200_000 default gas limit * 0.1 utia per gas = 20_000 utia per transaction.

So is this saying that most transactions will consume less than 20_000 utia?